### PR TITLE
Crowdsourcing feedback session comments

### PR DESCRIPTION
**Pull request by brahmaneya**
**Test Plan**
tox -e crowdsourcing passes

### CODE REVIEW

File: crowdsourcing/crowdsourcing_tutorial.py [hunk 1]
```diff
@@ -143,7 +148,7 @@ def get_worker_labeling_function(worker_id, f):
 # %%
 from snorkel.labeling.analysis import LFAnalysis
 
-LFAnalysis(L_dev, lfs).lf_summary(Y_dev).head(10)
+LFAnalysis(L_dev, lfs).lf_summary(Y_dev).head(8)
```
Comment (vincentschen) [inline crowdsourcing/crowdsourcing_tutorial.py]: ? seems arbitrary

File: crowdsourcing/crowdsourcing_tutorial.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # %% [markdown]
 # # Crowdsourcing tutorial
-# In this tutorial, we'll provide a simple walkthrough of how to use Snorkel to resolve conflicts
-# in a noisy, hybrid dataset for a sentiment analysis task.
-# We have crowdsourced labels for a portion of the training dataset, and combine these
-# with heuristic labeling functions to increase the number of training labels we have.
+# In this tutorial, we'll provide a simple walkthrough of how to use Snorkel alongside Crowdsourcing to generate labels for a sentiment analysis task.
```
Comment (vincentschen) [inline crowdsourcing/crowdsourcing_tutorial.py]: nit: I don't think we should capitalize Crowdsourcing, because it's a common technique

File: crowdsourcing/crowdsourcing_tutorial.py [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
+# We have crowdsourced labels for a portion of the training dataset.
```
Comment (vincentschen) [inline crowdsourcing/crowdsourcing_tutorial.py]: can we specify what the "portion" is?

File: crowdsourcing/crowdsourcing_tutorial.py [hunk 4]
```diff
@@ -156,10 +161,9 @@ def get_worker_labeling_function(worker_id, f):
 # %% [markdown]
 # ### Additional labeling functions
 #
-# We can mix the crowd worker labeling functions with labeling
+# To improve coverage and get more training labels, we can mix the crowd worker labeling functions with labeling
```
Comment (vincentschen) [inline crowdsourcing/crowdsourcing_tutorial.py]: coverage of what?

File: crowdsourcing/crowdsourcing_tutorial.py [hunk 5]
```diff
@@ -290,3 +294,11 @@ def encode_text(text):
 sklearn_model.fit(train_vectors, probs_to_preds(Y_train_prob))
 
 print(f"Accuracy of trained model: {sklearn_model.score(test_vectors, Y_test)}")
+
+# %% [markdown]
+# ## Summary
+#
+# In this tutorial, we accomplished the following:
+# * We showed how Snorkel can handle Crowdsourced labels, combining them with other programmatic LFs to improve coverage.
+# * We showed how the Label Model learns to combine inputs from crowd workers and other LFs by appropriately weighting them to generate high quality probabilistic labels.
```
Comment (vincentschen) [inline crowdsourcing/crowdsourcing_tutorial.py]: nit: LabelModel one word

File: crowdsourcing/crowdsourcing_tutorial.py [hunk 6]
```diff
@@ -156,10 +161,9 @@ def get_worker_labeling_function(worker_id, f):
 # %% [markdown]
 # ### Additional labeling functions
 #
-# We can mix the crowd worker labeling functions with labeling
+# To improve coverage and get more training labels, we can mix the crowd worker labeling functions with labeling
 # functions of other types.
-# We'll use a few varied approaches and use the label model learn
-# how to combine their values.
+# For example, we can use [TextBlob](https://textblob.readthedocs.io/en/dev/index.html), a tool that provides a pretrained sentiment analyzer. We run TextBlob on our tweets and create some simple LFs that threshold its polarity score.
```
Comment (henryre) [inline crowdsourcing/crowdsourcing_tutorial.py]: note that we picked them in a similar way to the spam tutorial

File: crowdsourcing/crowdsourcing_tutorial.py (hunk 1 above)
Comment (brahmaneya) [reply crowdsourcing/crowdsourcing_tutorial.py]: The 9th entry is a worker who has no labels in the dev set (but does have labels in the train set). This confused some people in the feedback session. The alternative is to add a comment explaining this, but it seemed like it would be less distracting to just not show that entry.

File: crowdsourcing/crowdsourcing_tutorial.py (hunk 3 above)
Comment (henryre) [reply crowdsourcing/crowdsourcing_tutorial.py]: add a little narrative here. common practice to send some amount of data to crowdworkers, in the meantime can gather more unlabeled data, want to leverage but not submit a new crowdworking job

File: crowdsourcing/crowdsourcing_tutorial.py [hunk 7]
```diff
@@ -290,3 +294,11 @@ def encode_text(text):
 sklearn_model.fit(train_vectors, probs_to_preds(Y_train_prob))
 
 print(f"Accuracy of trained model: {sklearn_model.score(test_vectors, Y_test)}")
+
+# %% [markdown]
+# ## Summary
+#
+# In this tutorial, we accomplished the following:
+# * We showed how Snorkel can handle Crowdsourced labels, combining them with other programmatic LFs to improve coverage.
```
Comment (henryre) [inline crowdsourcing/crowdsourcing_tutorial.py]: lowercase

File: crowdsourcing/crowdsourcing_tutorial.py [hunk 8]
```diff
@@ -290,3 +294,11 @@ def encode_text(text):
 sklearn_model.fit(train_vectors, probs_to_preds(Y_train_prob))
 
 print(f"Accuracy of trained model: {sklearn_model.score(test_vectors, Y_test)}")
+
```
Comment (henryre) [inline crowdsourcing/crowdsourcing_tutorial.py]: small comment about accuracy. almost as good as having crowd labels in perpetuity for all unseen examples

Comment (brahmaneya) [reply crowdsourcing/crowdsourcing_tutorial.py]: Will add. I didn't originally do this because 86% vs 92% is a pretty big difference, so I don't want it to look like we're over-claiming.

Comment (henryre) [reply crowdsourcing/crowdsourcing_tutorial.py]: it's pretty good given that we have no crowdsourcing labels for test! definitely acknowledge that it's not perfect

File: crowdsourcing/crowdsourcing_tutorial.py (hunk 1 above)
Comment (vincentschen) [reply crowdsourcing/crowdsourcing_tutorial.py]: I see—super nit, but maybe we can just show `.head(5)`? Seems less arbitrary than 8

Comment (brahmaneya) [reply crowdsourcing/crowdsourcing_tutorial.py]: Done
